pages/linux/stress.md

```diff
@@ -0,0 +1,19 @@
+# stress
+
+> A tool to stress test CPU, memory, and IO on a Linux system.
+
+- Spawn N workers to stress test CPU:
+
+`stress -c {{N}}`
+
+- Spawn N workers to stress test IO and timeout after M seconds:
+
+`stress -i {{N}} -t {{M}}`
+
+- Spawn N workers to stress test memory(each worker malloc 256M bytes):
```

Comment:
```suggestion
- Spawn N workers to stress test memory (each worker malloc 256M bytes):
```
Also maybe change `malloc` to something more explanative?

Reply:
I agree, but I cannot find another word to explain. Any idea?

Reply:
allocates